controllers/operatorpolicy_controller.go

```diff
@@ -55,6 +55,8 @@ const (
 	ManagedByAnnotation    string = ManagedByLabel
 	ClusterNameLabel       string = "policy.open-cluster-management.io/cluster-name"
 	ClusterNamespaceLabel  string = "policy.open-cluster-management.io/cluster-namespace"
+	SubLockAnnotation      string = "operatorpolicy.policy.open-cluster-management.io/namechange"
```

Comment:
I think I'd prefer if the user got a noncompliant message on the subscription if the name or namespace changed and that the solution is to create a new or recreate the old OperatorPolicy or uninstall the existing operator.

Having this annotation override is more confusing.